webpack.dev.config.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
```

Comment:
pls add for dev mode devtool property that controls if and how source maps are generated, it will help you debug your code on development stage

https://webpack.js.org/configuration/devtool/